molecule/playbooks/prepare.yml

```diff
@@ -13,6 +13,12 @@
       shell: "sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-Linux-*"
       when: ansible_distribution == "CentOS" and ansible_distribution_major_version == "8"
 
+    - name: install GPG key for epel 7
+      rpm_key:
+        state: present
+        key: https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-7
+      when: ansible_os_family == "RedHat" and ansible_distribution_major_version == "7"
```

Comment:
Do we need to add the key for all the RedHat 7  or for Amazon2?

Reply:
Fixed , added key for Amazon2 only